sw/device/lib/testing/usb_testutils_controlep.h

```diff
@@ -25,6 +25,8 @@ typedef enum usb_testutils_ctstate {
   kUsbTestutilsCtStatOut,     // Waiting for OUT status stage
   kUsbTestutilsCtAddrStatIn,  // Queued status stage, waiting ack.
                               // After which, set dev_addr
+  kUsbTestutilsCtCfgStatIn,   // Queued status stage, waiting ack.
+                              // After which, set usb_config
```

Comment:
Nit: IIUC the only difference between the two is that one is waiting for the address the other one for the config to be staged. This could be made more clear in the comment but it's not critical I think.

Reply:
Thanks; it's a good point. The commenting in the USB testutils code has always been a bit sparse and not as helpful as it could be to people who are not yet familiar with the protocol specification. I've elaborated the comments.

Reply:
Thanks @alees24 , now it's very clear :-)